blockmanager.go

```
@@ -697,12 +933,16 @@ func (b *blockManager) getCFHeadersForAllPeers(height uint32,
 	// Create the map we're returning.
 	headers := make(map[string]*wire.MsgCFHeaders)
 
-	// Get the header we expect at the last height represented by the
-	// checkpoint number passed to us.
-	stopHeader, err := b.server.BlockHeaders.FetchHeaderByHeight(
-		height + wire.MaxCFHeadersPerMsg - 1)
-	if err != nil {
-		return headers
+	// Get the header we expect at either the tip of the block header store
+	// or at the end of the maximum-size response message, whichever is
+	// larger.
+	stopHeader, stopHeight, err := b.server.BlockHeaders.ChainTip()
```

Ignoring the error here.